pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -412,20 +412,21 @@ func proxyMapToEnvVars(proxyConfig map[string]string) []corev1.EnvVar {
 	return envVars
 }
 
-func gracefulTerminationDurationFromConfig(operatorSpec *operatorv1.StaticPodOperatorSpec) (int, error) {
+func gracefulTerminationDurationFromConfig(config map[string]interface{}) (duration int, err error) {
```

Comment:
no named return values.  Too many accidents in kube, so we disallow it in the vast majority of cases.